docs(tutorial): raid on em server MTA-2108 (#756)
scaleway/docs-content: tutorials/remove-raid-elastic-metal/index.mdx

---
meta:

>   title: Custom Raid Configuration on Elastic Metal

Reviewer: ```suggestion
  title: Custom RAID Configuration on Elastic Metal
```

>   description: Custom Raid Configuration on Elastic Metal

Reviewer: ```suggestion
  description: Custom RAID Configuration on Elastic Metal
```

content:

>   h1: Custom Raid Configuration on Elastic Metal

Reviewer: ```suggestion
  h1: Custom RAID Configuration on Elastic Metal
```

>   paragraph: Custom Raid Configuration on Elastic Metal

Reviewer: ```suggestion
  paragraph: Custom RAID Configuration on Elastic Metal
```

tags: RAID elastic-metal
categories:
  - compute
  - elastic-metal
dates:
  validation: 2022-08-02
  posted: 2022-09-02
---

> Each Elastic Metal server is using a RAID1 configuration after installation from the Scaleway console. If you want to change the RAID configuration of the server, you can modify the RAID array using [rescue mode](/compute/elastic-metal/how-to/use-rescue-mode/).

Reviewer: ```suggestion
Each Elastic Metal server uses a RAID1 configuration after installation from the Scaleway console. If you want to change the RAID configuration of the server, you can modify the RAID array using [rescue mode](/compute/elastic-metal/how-to/use-rescue-mode/).
```

## Removing the standard RAID configurion

1. Boot server in [rescue mode](/compute/elastic-metal/how-to/use-rescue-mode/) from the Scaleway console.

> 2. Login to the server using the `root` account:

Reviewer: ```suggestion
2. Log in to the server using the `root` account:
```

    ```
    ssh root@<your_elastic_metal_ip>
    ```

> 3. Visualize the existing mdadm RAID devices by running the following command:

Reviewer 2: ```suggestion
3. Visualize the existing `mdadm` RAID devices by running the following command:
```

    ```
    cat /proc/mdstat
    ```

    An output as follows displays:

    ```sh
    root@elastic-metal:~# cat /proc/mdstat
    Personalities : [linear] [multipath] [raid0] [raid1] [raid6] [raid5] [raid4] [raid10]
    md126 : active (auto-read-only) raid1 sda4[0] sdb4[1]
        998311936 blocks super 1.2 [2/2] [UU]
        bitmap: 0/8 pages [0KB], 65536KB chunk

    md127 : active (auto-read-only) raid1 sda3[0] sdb3[1]
        613376 blocks super 1.2 [2/2] [UU]

    unused devices: <none>
    ```

    <Message type="note">

    In the example above you find two devices `md126` and `md127`, both in RAID1 mode using the disks `sda` and `sdb`.

    </Message>

> 4. Stop the mdadm devices using the `mdadm --stop` command:

Reviewer 2: ```suggestion
4. Stop the `mdadm` devices using the `mdadm --stop` command:
```

    ```sh
    root@elastic-metal:~# mdadm --stop /dev/md126
    mdadm: stopped /dev/md126
    root@elastic-metal:~# mdadm --stop /dev/md127
    mdadm: stopped /dev/md127
    ```

> 5. Remove the mdadm devices using the `mdadm --remove` command: 

Reviewer 2: ```suggestion
5. Remove the `mdadm` devices using the `mdadm --remove` command: 
```

    ```sh
    root@elastic-metal:~# mdadm --remove /dev/md126
    root@elastic-metal:~# mdadm --remove /dev/md127
    ```

    <Message type="tip">

    When running the command above, the following error may appear: 

    `mdadm: error opening md127: No such file or directory`

>     This is normal behavior. Ignore it, and go ahead with the newt step.

Reviewer 2: ```suggestion
    This is normal behavior. Ignore it, and go ahead with the next step.
```

    </Message>

6. Remove the Superblocks on the disks using the `mdadm --zero-superblock` command:

```
root@elastic-metal:~# mdadm --zero-superblock /dev/sda4 /dev/sdb4
root@elastic-metal:~# mdadm --zero-superblock /dev/sda3 /dev/sdb3
```

> 6. Verify the RAID configuration has been removed using the `lsblk` and `cat /proc/mdstat` commands: 

Reviewer 3: ```suggestion
7. Verify the RAID configuration has been removed using the `lsblk` and `cat /proc/mdstat` commands: 
```
